.github/workflows/compute.yml

```diff
@@ -16,31 +16,25 @@ on:
       ref:
         description: "Ref"
       signature:
-        description: "The kernel signature"
+        description: "Signature sent from the Kernel"
+      command:
+        description: "Command"
 
 jobs:
   compute:
-    name: "plugin name"
+    name: "Pull review"
     runs-on: ubuntu-latest
     permissions: write-all
+    environment: ${{ github.ref == 'refs/heads/main' && 'main' || 'development' }}
+
     env:
-      SUPABASE_URL: ${{ secrets.SUPABASE_URL }}
-      SUPABASE_KEY: ${{ secrets.SUPABASE_KEY }}
+      KERNEL_PUBLIC_KEY: ${{ secrets.KERNEL_PUBLIC_KEY }}
+      PLUGIN_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why are you calling it plugin GitHub token? The standard is just calling it GitHub token. 

Why are you passing it in?